src/discussions/comments/CommentsView.jsx

```diff
@@ -76,7 +76,8 @@ function DiscussionCommentsView({
         {comments.map(comment => (
           <Comment comment={comment} key={comment.id} postType={postType} />
         ))}
-
+        {!!comments.length
```

Comment:
Why the trailing !! ?

Reply:
because it adds length count in dom. it is to change it to boolean